src/libespeak-ng/phoneme.h

```diff
@@ -250,7 +250,6 @@ phoneme_add_feature(PHONEME_TAB *phoneme,
 // one for the current voice
```

Comment:
I think, there is also misleading comment before that code:
// Several phoneme tables may be loaded into memory. phoneme_tab points to
// one for the current voice
looks like should be: n_phoneme_tab points to..

Reply:
See this:
https://github.com/espeak-ng/espeak-ng/blob/30d5735db5aaea8d27b68bdf8309c72f5fc8e1e9/src/libespeak-ng/phoneme.h#L227-L228

I think N_PHONEME_TAB in `PHONEME_TAB *phoneme_tab[N_PHONEME_TAB]` means it is a single phoneme table, whereas a `PHONEME_TAB_LIST phoneme_tab_list[N_PHONEME_TABS]` is a list of phoneme tables. n_phoneme_tables is probably an index into a phoneme list.

n_phoneme_tab should also be a parameter instead of global.